cmd/main.go

```diff
@@ -38,7 +39,7 @@ var Version = "undefined"
 var (
 	portPtr          = kingpin.Flag("port", "port gRPC server listens on").Default("55912").OverrideDefaultFromEnvar("NEW_RELIC_PORT").Short('p').Int32()
 	clusterNamePtr   = kingpin.Flag("cluster-name", "Name of cluster where metrics come from").OverrideDefaultFromEnvar("NEW_RELIC_CLUSTER_NAME").String()
-	debugPtr         = kingpin.Flag("debug", "enable debug logging").OverrideDefaultFromEnvar("NEW_RELIC_DEBUG").Short('d').Bool()
+	logLevelPtr      = kingpin.Flag("log-level", "set logging level").OverrideDefaultFromEnvar("NEW_RELIC_LOG_LEVEL").Default("error").Enum(log.Levels()...)
```

Comment:
Should update the example command in DEVELOPMENT.md to use this flag instead of `--debug`

Reply:
It would be nice to know what other values `--log-level` can be set to other than `error`

Reply:
Actually the error message is pretty clear if you specify anything unexpected, I'm fine leaving the help message as is or adding the allowed values to it.
```
main: error: enum value must be one of fatal,none,debug,info,warn,error, got 'help', try --help
```

Reply:
@kdef-nr : w/r/t printing the `--log-level` options I agree. It is something inherent in `kingpin` unfortunately. I was thinking of transitioning to `cobra` (which Istio uses), but it seemed like a larger thing we can tackle if deemed necessary later on.